docs/user/openstack/README.md

```diff
@@ -33,7 +37,43 @@ clouds:
       auth_url: 'https://10.10.14.22:5001/v2.0'
 ```
 
-* Swift must be enabled.  The user must have `swiftoperator` permissions and
+If you choose to use environment variables in place of a clouds.yaml, or along side it, consult the following doccumentation:
+https://www.terraform.io/docs/providers/openstack/#configuration-reference
+
+
+## OpenStack Requirements
+
+### Recommended Minimums
+
+In order to run the latest version of the installer in OpenStack, at a bare minimum you need the following quota to run a *default* cluster. While it is possible to run the cluster with fewer resources than this, it is not recommended. Certian edge cases, such as deploying [without FIPs](#without-floating-ips), or deploying with an [external loadbalancer](#using-an-external-load-balancer) are documented below, and are not included in the scope of this recomendation.
+
+   * OpenStack Quota
+     * Floating IPs: 3
```

Comment:
This is really 2 floating IPs.

Reply:
no, it's three. One floating IP is taken up by the router itself.

Reply:
Or rather, one IP from the floating range is taken up by the router even though it doesn't show up in `openstack floating ip list`. But your range needs at least 3 IPs.

Reply:
Hmm ok, but does it count as part of the FIP quota?

Reply:
Hmm, that's a good question. I'm not sure, but since it comes from the same range, having enough quota (backed by FIP count) should always work here.